test/run.sh

```diff
@@ -77,13 +72,25 @@ if ! command -v docker; then
   exit 1
 fi
 
+KIND_BIN="$BIN_DIR/kind"
+if ! command -v $KIND_BIN; then
+  echo "'kind' not available in '$BIN_DIR', please run 'make depend'"
+  exit 1
+fi
+
+HELM_BIN="$BIN_DIR/helm"
+if ! command -v $HELM_BIN; then
+  echo "'helm' not available in '$BIN_DIR', please run 'make depend'"
+  exit 1
+fi
```

Comment:
suggestion (non-blocking):  Is it worth adding a `make run-test` (name obviously not importantly) so that the happy path is to interact directly with a single PHONY make target which can provision the dependencies for you?

Reply:
Is that not what we already have? https://github.com/cert-manager/csi-driver/blob/0eec76db51d2607ae1dd3176fcfb4d651586451c/Makefile#L67

Reply:
Ahh :facepalm: Yeah, dumb comment by me, sorry!